halibot/message.py

```diff
@@ -26,12 +26,26 @@ def makeOrigin(self):
 
 class Message():
 
+	class Author(str):
+		def __init__(self, a, id=""):
+			self.id = id
+			self.name = str(a) if a else ""
+		def __str__(self):
+			return self.name
+		def __repr__(self):
+			return self.name
```

Comment:
This implementation of `__repr__` does not follow the [python documentation](https://docs.python.org/3.5/reference/datamodel.html#object.__repr__), which is either to return a python expression that can be used to reconstruct the object or a useful description inside waka-wakas.